components/AnchorCollapse/src/index.scss

```diff
@@ -0,0 +1,96 @@
+.denhaag-anchor-collapse {
+  --denhaag-anchor-collapse-box-shadow: inset 0 -1px 0 var(--denhaag-anchor-collapse-border-color);
```

Comment:
I am not a fan of box-shadows used as borders. 
If shadows are needed for aesthetic purpose besides the default styling, I'd use box-shadows.
But I guess this is a preferred thing.

Reply:
edited